docs/basics/part-8a-using-earthly-in-your-current-ci.md

```diff
@@ -11,10 +11,17 @@ For more information on how to use Earthly in other CIs such as GitLab, Jenkins,
 To use Earthly in a CI, you typically encode the following steps in your CI's build configuration:
 
 1. Download and install Earthly
```

Comment:
Should step 1 be signing up for Earthly Cloud?

Reply:
This is more about what the CI script does, not what you do as the user. Not sure how to make that a little more obvious.